dashboard/app/views/pd/workshop_daily_survey/new_general_foorm.haml

```diff
@@ -0,0 +1,6 @@
+- content_for(:head) do
+  = stylesheet_link_tag 'css/pd', media: 'all'
+  %script{src: webpack_asset_path('js/pd/workshop_daily_survey/new_general_foorm.js'), data: @script_data}
+
+#application-container
+  -# populated by React
```

Comment:
:open_mouth: :exclamation: :1st_place_medal: Why do we not always have this comment.

Reply:
copied from some other smart view!